roles/openshift_cli/meta/main.yml

```diff
@@ -12,6 +12,6 @@ galaxy_info:
   categories:
   - cloud
 dependencies:
-- { role: openshift_docker, upgrading: "{{ upgrading | default(False) }}" }
```

Comment:
Knowing what I know now about how we pass vars in from playbooks, and then they're visible to all roles, is an explicit passthrough of this variable needed here? I'm wondering now if it's necessary at all...